integration/cri-o/crio_skip_tests.sh

```diff
@@ -5,6 +5,10 @@
 # SPDX-License-Identifier: Apache-2.0
 #
 
+# TODO: the list of files is hardcoded for now, as we don't have listed all tests
+# that need to be skipped, but ultimately we will want to use *.bats
+declare -a bats_files_list=("ctr.bats" "default_mounts.bats" "hooks.bats" "image_remove.bats" "image_volume.bats" "infra_ctr_cpuset.bats" "profile.bats" "reload_config.bats" "reload_system.bats" "runtimeversion.bats" "sanity_checks.bats" "selinux.bats" "shm.bats" "stats.bats" "status.bats" "version.bats" "inspecti.bats" "cgroups.bats" "command.bats" "config.bats" "config_migrate.bats" "crio-wipe.bats" "ctr_seccomp.bats" "devices.bats" "drop_infra.bats" "image.bats" "inspect.bats" "metrics.bats" "namespaces.bats" "network.bats" "network_ping.bats" "pod.bats" "restore.bats" "shm_size.bats" "timeout.bats" "workloads.bats" "userns_annotation.bats")
```

Comment:
Please can you fold this long long (and ensure the list of tests is sorted alphabetically):

```suggestion
declare -a bats_files_list=(
    "ctr.bats"
    "default_mounts.bats"
    "hooks.bats"
    # ...
    )
```

Reply:
Agree with @jodh-intel for the formatting.
Note that this list will be added by #4434 - you will need to rebase after it is merged.

Reply:
Thanks, I folded long list alphabetically :) , I will re-base it once #4434 is merged.